Organizing getting started redirects and adding search terms (#2510)
fishtown-analytics/docs.getdbt.com: website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md

title: Build your first models
id: build-your-first-models
description: "Now that you've set up the starter project, you can get to the fun part — building models!"
keywords: [projects, create a project, dbt project]
---

> Now that you set up your sample project and had a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.

Author: ```suggestion
After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
```

title: Build your first models
id: build-your-first-models
description: "Now that you've set up the starter project, you can get to the fun part — building models!"
keywords: [projects, create a project, dbt project, get started with Cloud]
---

> After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.

Author: ```suggestion
After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will learn how to take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
```
